packages/block-library/src/embed/wp-embed-preview.js

```diff
@@ -58,8 +58,15 @@ export default function WpEmbedPreview( { html } ) {
 				return;
 			}
 
-			const focusEvent = new FocusEvent( 'focus', { bubbles: true } );
-			activeElement.dispatchEvent( focusEvent );
+			// The focus event should *not* bubble:
+			// https://w3c.github.io/uievents/#event-type-focus
+			activeElement.dispatchEvent( new FocusEvent( 'focus' ) );
+
+			// The focusin event *should* bubble:
+			// https://w3c.github.io/uievents/#event-type-focusin
+			activeElement.dispatchEvent(
+				new FocusEvent( 'focusin', { bubbles: true } )
+			);
```

Comment:
Can't we just do this?
```suggestion
			activeElement.focus();
```

Reply:
This was first introduced in 5730. I didn't think it would work, but it does. The right window seems to stay focussed and I can keyboard navigate as you would expect.